src/components/App/App.js

```diff
@@ -18,7 +20,17 @@ class App extends Component {
 	}
 
 	confirmMatch(index, img) {
-		if (!this.state.prediction) this.setState({prediction: {index, img}});
+		this.setState(
+			prevState =>
+				prevState.prediction ? null : {prediction: {index, img}}
+		);
+	}
+
+	componentDidMount() {
+		loadUsers(users => {
+			if (users) this.setState({users: users});
```

Comment:
Nitpick until linter is setup, we could take advantage of object shorthand `this.setState({ users })`